test/functional/rpc_signrawtransaction.py

```diff
@@ -1,8 +1,8 @@
 #!/usr/bin/env python3
-# Copyright (c) 2015-2017 The Bitcoin Core developers
+# Copyright (c) 2015-2018 The Bitcoin Core developers
 # Distributed under the MIT software license, see the accompanying
-# file COPYING or http://www.opensource.org/licenses/mit-license.php.
-"""Test transaction signing using the signrawtransaction* RPCs."""
+# file COPYING or https://opensource.org/licenses/mit-license.php.
+"""Test transaction signing using the signrawtransaction* RPC."""
```

Comment:
Why this change?

Reply:
My mistake. Glitch in my workflow after new rebase experience. Will fix soon. If I might ask: currently I replace old commits in this PR/my branch with hard resets and `push -f`, to keep changes small and honor the `scripted_diff` checks (except in last case, my apologies) - is that a correct thing to do?

Reply:
Jup, correct. You can also check the scripted diffs with the `./contrib/devtools/commit-script-check.sh` script.